src/js/components/AppStatusComponent.jsx

```diff
@@ -13,7 +13,8 @@ var statusClassNameMapping = {
   [AppStatus.DEPLOYING]: "deploying",
   [AppStatus.SUSPENDED]: "suspended",
   [AppStatus.DELAYED]: "delayed",
-  [AppStatus.WAITING]: "waiting"
+  [AppStatus.WAITING]: "waiting",
+  [AppStatus.WAITING_FOR_DECISION]: "waiting-for-action"
```

Comment:
The naming naming of the key should at least aligned with the value.
`AppStatus.WAITING_FOR_ACTION`
or maybe
`AppStatus.WAITING_FOR_USER_ACTION`